app/Http/Controllers/Auth/AuthController.php

```diff
@@ -38,40 +36,38 @@ public function getCheckpoint()
 
     public function getIndex()
     {
+        $state = str_random(5);
```

Comment:
Has little but considerable probability to create the same string when lots of users are in. What happens then?